tests/run-make/issue-14500/rmake.rs

```diff
@@ -0,0 +1,26 @@
+// Test to make sure that reachable extern fns are always available in final
+// productcs, including when LTO is used.
+
+// In this test, the `foo` crate has a reahable symbol,
+// and is a dependency of the `bar` crate. When the `bar` crate
+// is compiled with LTO, it shouldn't strip the symbol from `foo`, and that's the
+// only way that `foo.c` will successfully compile.
+// See https://github.com/rust-lang/rust/issues/14500
+
+//@ ignore-cross-compile
+
+use run_make_support::{cc, extra_c_flags, run, rustc, tmp_dir};
+
+fn main() {
+    let libbar_path = tmp_dir().join("libbar.a");
```

Comment:
I don't recall exactly, do we have a static library name helper? If not, can you add one?

Reply:
There is already one! I should use it.